apps/fluent-tester/src/RNTester/TestComponents/Icon/IconTest.tsx

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+import * as React from 'react';
+import { View } from 'react-native';
+import { Text } from '@fluentui-react-native/text';
```

Comment:
This can be imported from '@fluentui/react-native' instead.

Reply:
Ok importing from there instead.